application/org.openjdk.jmc.flightrecorder.flameview/src/main/resources/jsjmclibs/flameviewColoring.js

```diff
@@ -0,0 +1,92 @@
+String.prototype.hashCode = function () {
+    var hash = 0;
+    if (this.length === 0) return hash;
+    for (var i = 0; i < this.length; i++) {
+        var char = this.charCodeAt(i);
+        hash = ((hash << 5) - hash) + char;
+        hash = hash & hash;
+    }
+    return hash;
+};
+
+const rootPackageColor = "darkred";
+const invalidPackageColor = "snow";
+const packageJavaValueLMax = 80;
+const packageJavaValueL = 65;
+const packageJavaHslHSValues = [184, 10];
+const packageSunValueLMax = 54;
+const packageSunValueL = 31;
+const packageSunHSValues = [211, 21];
+const packageRestValueHMax = 360;
+const packageRestValueH = 0;
+const packageRestSLValues = [42, 53];
+const packageConsideredDepth = 3;
+const packageMarkerJava = "java";
+const packageMarkerSun = "sun";
+const packageMarkerRest = "rest";
+const packagesIdentifierMap = new Map().set("java.", packageMarkerJava).set("sun.", packageMarkerSun)
+    .set("com.sun.", packageMarkerSun);
+const packageColorMap = new Map().set("", rootPackageColor);
+
+const colorByPackage = function (p) {
+    if (p === undefined) {
+        return invalidPackageColor;
+    } else {
+        const packageNameStrip = stripPackageName(p);	
+        const packageSelectedColor = packageColorMap.get(packageNameStrip);
+        if (packageSelectedColor === undefined) {
+            const packageMarkerSelected = getPackageMarker(packageNameStrip);
+
```

Comment:
Formatting looks funny here.

Reply:
I've not spot anything unusual.  IDE gives me such formatting by default

Reply:
Indentation level looks like it is different for packageMarkerSelected and packageNameStripHash, but it should not be. I would also remove the new line.

Reply:
corrected